taqueria-config.ts

```diff
@@ -51,6 +55,11 @@ export const defaultConfig: Config.t = Config.create({
 			rpcUrl: 'https://ghostnet.ecadinfra.com',
 			protocol: 'PtKathmankSpLLDALzWw7CGD2j2MtyveTwboEYokqUCP4a1LxMg',
 		},
+		mainnet: {
+			label: 'mainnet',
+			rpcUrl: 'https://mainnet.api.tez.ie',
+			protocol: 'PtKathmankSpLLDALzWw7CGD2j2MtyveTwboEYokqUCP4a1LxMg',
```

Comment:
@mweichert just want to confirm that protocol here can be removed right? I recall that it is ignored anyways, for non-sandbox networks.